runtime/templates/qemu/bare64.mk

```diff
@@ -0,0 +1,39 @@
+ISP_PREFIX ?= $(HOME)/.local/isp/
```

Comment:
We don't need duplicate files if only a few fields change (eg. the compiler)

Reply:
Consolidated makefiles.